README.md

```diff
@@ -16,9 +16,6 @@
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1320/badge)](https://bestpractices.coreinfrastructure.org/projects/1320)
 ![Code style](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)
 [![Follow on Twitter](https://img.shields.io/twitter/follow/MaterialUI.svg?label=follow+Material-UI)](https://twitter.com/MaterialUI)
-[![PeerDependencies](https://img.shields.io/david/peer/mui-org/material-ui/next.svg)](https://david-dm.org/mui-org/material-ui/next?type=peer&path=packages/material-ui)
```

Comment:
Is there a reason we don't use david-dm directly? As far as I can work out the reason why the badges are broken is img.shields.io doesn't support branches.

Working: https://img.shields.io/david/peer/mui-org/material-ui.svg
Not working: https://img.shields.io/david/peer/mui-org/material-ui/next.svg

Working: https://david-dm.org/mui-org/material-ui/next/peer-status.svg?path=packages/material-ui

Reply:
Oh, right! I didn't realise these were coming from separate services.